DBConnect.cc

```diff
@@ -54,4 +58,310 @@ void ConnectToMongoDB() {
     mongoc_cleanup();
 }
 
+bool SaveLayoutToDB(const std::string& name, const std::string& json_string) {
+    const char* uri_string = "mongodb://localhost:27017";
```

Comment:
do you think we should make the mongo address an optional argument? If servers run multiple MongoDB instances, this port might be unavailable. In addition, if this DB is running on a separate node, we'd need to provide the address as well

Reply:
Should this not be stored in a configuration file, rather than hardcoded?

Reply:
The backend is not using a config file currently. We could add one again, though it is something else to manage. I could put this as a command like option. Quite a few server programs have configs managed by things like initd which passed them arguments.

Reply:
I think a command line option is ok